unstructured/staging/base.py

```diff
@@ -176,11 +178,14 @@ def convert_to_csv(elements: List[Element]) -> str:
     return convert_to_isd_csv(elements)
 
 
-def convert_to_dataframe(elements: List[Element], drop_empty_cols: bool = True) -> pd.DataFrame:
+@requires_dependencies(["pandas"])
+def convert_to_dataframe(elements: List[Element], drop_empty_cols: bool = True):
```

Comment:
Same here, I think we can still include the typing by making the type a string, i.e. .
```suggestion
def convert_to_dataframe(elements: List[Element], drop_empty_cols: bool = True) -> "pd.DataFrame":
```

Reply:
Oh! I didn't realize you could do that. Will update